### ISSUE TITLE
Cannot set borderRadius style for OTSubscriber or OTPublisher

### ISSUE THREAD

**Issue by sunweiyang** [opentok/opentok-react-native#174]
# Bug Report

**Current behavior**
Styles generally work for OT* components, but the `borderRadius` style is ignored if specified. For example, opentok-react-native users cannot create circular publishers or subscribers like the following:

![profile1](https://user-images.githubusercontent.com/12490625/45911201-36c6b800-bdc5-11e8-8ce7-31986f9a865f.png)

**Steps to reproduce**
Use `borderRadius` styles in OTPublisher and/or OTSubscriber. For example:
```
<OTSession apiKey="your-api-key" sessionId="your-session-id" token="your-session-token">
  <OTPublisher style={{ width: 100, height: 100, borderRadius: 50 }}/>
  <OTSubscriber style={{ width: 100, height: 100, borderRadius: 50 }} />
</OTSession>
```

**What is the current *bug* behavior?**
If `borderRadius` style is used with opentok-react-native components, the app behaves as if the `borderRadius` style is never specified.

**What is the expected *correct* behavior?**
The `borderRadius` style should properly occlude the rectangular subscriber/publisher render.

**Comment by jgutierrez11** [comment]
What I did is put the OTPublisher inside a View with overflow set to 'hidden', hope it helps

**Comment by sunweiyang** [comment]
@jgutierrez11 Thanks; however this doesn't work on Android. This is probably related to #162.

**Comment by msach22** [comment]
@sunweiyang @ggoldens We plan on wrapping the `OTSubscriberView` component with a `View` so you should be able to add this then.

We'll update you when it goes out.

**Comment by enricop89** [comment]
Should be fixed with `0.11.2` version @sunweiyang . Doc here [Custom Rendering Subscriber](https://github.com/opentok/opentok-react-native/blob/master/docs/OTSubscriber.md#custom-rendering-of-streams)

**Comment by merovingienne** [comment]
Has anyone been able to add a border radius to the `OTPublisher`?
